### PR TITLE
Add basic launch instance with ansible tutorial

### PR DESCRIPTION
**Pull request by squidboylan**

### CODE REVIEW

**Comment by justinknierim** [comment]
Looks good to me.

File: source/dreamcompute/tutorials/launch-an-instance-with-ansible.rst [hunk 1]
```diff
@@ -0,0 +1,151 @@
+======================================================
+How to launch an server with Ansible on DreamCompute
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/launch-an-instance-with-ansible.rst]: there is a typo... Wouldn't it look better with title:

 How to launch a DreamCompute cloud server with Ansible

?

**Comment by justinknierim** [comment]
I think I prefer "on DreamCompute" at the end myself from a flow standpoint.  My $0.02.  Good catch on the "an server" typo.

File: source/dreamcompute/tutorials/launch-an-instance-with-ansible.rst [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+Ansible is a configuration management tool with built in OpenStack support.
+Which means it's easy to use it to deploy servers and manage them on
+DreamCompute using Ansible. In this tutorial we're going to explain how to
+install Ansible and use it to launch a simple server on DreamCompute.
+
+Installation
+~~~~~~~~~~~~
+
+For this tutorial we're going to use Ubuntu 16.04 (Xenial Xerus) as our host to
+run ansible plays from. First we need to install a few packages.
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/launch-an-instance-with-ansible.rst]: aren't they called 'ansible playbooks'?

**Comment by squidboylan** [comment]
Oops, just changed it, I can change it back if we decide the other way was better though. Sounds like stef has more to add, but is busy right now

File: source/dreamcompute/tutorials/launch-an-instance-with-ansible.rst [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+On Ubuntu 16.04 that can be done with one simple command.
+
+.. code::
+
+    $ sudo apt-get install ansible python2.7 python-virtualenv python-pip
+
+Type in your password for sudo (If you have one), then hit **y** when it asks
+you for confirmation. It then will download some packages and install them.
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/launch-an-instance-with-ansible.rst]: i'd suggest to make this shorter, remove the password thing and **y** by passing -y to apt-get

$ sudo apt-get install -y ansible python2.7 python-virtualenv python-pip

We have to assume that people know how to use apt-get at a very minimum...

File: source/dreamcompute/tutorials/launch-an-instance-with-ansible.rst [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+Ansible is a configuration management tool with built in OpenStack support.
+Which means it's easy to use it to deploy servers and manage them on
+DreamCompute using Ansible. In this tutorial we're going to explain how to
+install Ansible and use it to launch a simple server on DreamCompute.
+
+Installation
+~~~~~~~~~~~~
+
+For this tutorial we're going to use Ubuntu 16.04 (Xenial Xerus) as our host to
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/launch-an-instance-with-ansible.rst]: I wonder if we should write this tutorial as multi-platform as possible... Maybe we document here how to install ansible on Linux and link to ansible docs to install on Mac OS and Windows

File: source/dreamcompute/tutorials/launch-an-instance-with-ansible.rst [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+On Ubuntu 16.04 that can be done with one simple command.
+
+.. code::
+
+    $ sudo apt-get install ansible python2.7 python-virtualenv python-pip
+
+Type in your password for sudo (If you have one), then hit **y** when it asks
+you for confirmation. It then will download some packages and install them.
+Next we need to install shade in a virtual environment as Ansible depends on
+it.
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/launch-an-instance-with-ansible.rst]: let's decide on how to spell ansible or Ansible...

**Pull request merged**